src/main/java/com/axiope/service/cfg/ProductionConfig.java

```diff
@@ -282,6 +283,8 @@ public TimeLimitedMemoryCache<License> licenseCache() {
   protected RemoteUserRetrievalPolicy remoteUserRetrievalPolicy() {
     if ("SAML".equals(deploymentSsoType)) {
       return new SAMLRemoteUserPolicy();
+    } else if ("openid".equals(deploymentSsoType)) {
```

Comment:
can we use `OPENID` capitalised as per `SAML` and `TEST`?

Reply:
`SAML` is capitalized as it's actually an acronym, `openid` doesn't have to be. But yeah I can switch to `equalsIgnoreCase`.